modules/ipi-install-establishing-communication-between-subnets.adoc

```diff
@@ -8,14 +8,20 @@
 
 In a typical {product-title} cluster setup, all nodes, including the control plane and worker nodes, reside in the same network. However, for edge computing scenarios, it can be beneficial to locate worker nodes closer to the edge. This often involves using different network segments or subnets for the remote worker nodes than the subnet used by the control plane and local worker nodes. Such a setup can reduce latency for the edge and allow for enhanced scalability. However, the network must be configured properly before installing {product-title} to ensure that the edge subnets containing the remote worker nodes can reach the subnet containing the control plane nodes and receive traffic from the control plane too.
 
-[IMPORTANT]
-====
-All control plane nodes must run in the same subnet. When using more than one subnet, you can also configure the Ingress VIP to run on the control plane nodes by using a manifest. See "Configuring network components to run on the control plane" for details.
+You can run control plane nodes in the same subnet or multiple subnets by configuring a user-managed load balancer in place of the default lod balancer. If you run a multiple subnet environment, you can reduce the risk of your {product-title} cluster from failing due to a hardware failure or a network outage. For more information, see "Services for a user-managed load balancer" and "Configuring a user-managed load balancer".
 
-Deploying a cluster with multiple subnets requires using virtual media.
+[NOTE]
+====
+Deploying a cluster with multiple subnets requires using virtual media, such as `redfish-virtualmedia` and `idrac-virtualmedia`.
 ====
 
-This procedure details the network configuration required to allow the remote worker nodes in the second subnet to communicate effectively with the control plane nodes in the first subnet and to allow the control plane nodes in the first subnet to communicate effectively with the remote worker nodes in the second subnet.
+A multiple subnet environment requires completion of the following key tasks:
```

Comment:
Most of these steps are specific to cross-subnet control plane, not necessarily all external loadbalancers. I think I need to figure out if we still want to support both cases because it's going to complicate the docs considerably.

Reply:
To follow up on this, we are going to stop documenting the old external loadbalancer flow where only external traffic went through it and just document the new fully user-managed loadbalancer where the internal one is disabled entirely. That should allow us to simplify the docs somewhat.

Reply:
One minor tweak to this is we should mention this is only _required_ if your control plane is on multiple subnets. You're not required to use a user-managed loadbalancer if you're just doing remote workers.